### PR TITLE
NEW TITLE SONG

### PR DESCRIPTION
**Pull request by yellowface7**
New theme - Fender Bender by Bad Snacks | Royalty free

Did I do this right? I sure hope I did. 

I have been working hard on a music overhaul. While working on it, I thought to myself, "hey, why not a new theme?". At the time, this server was brand new and the discord icon was just uploaded and I had no theme to go by at that time. I took a good look at the name of the server, then I took a good look at the icon, then I decided to go for a "sunset by the beach" theme to go by. I then plunged into the Youtube Audio Library and I came across this song. It was fresh new for it had a "new" tag next to it. I listened to it and liked it. I then thought to myself "man, all I need is some kind of beach sound effect". I dove into the code base for this server looking for something and sure enough, the sound effect titled "shore" would do. I opened up audacity, added this song, then added the sound effect and did some audio balancing. I was applying when the theme was decided. I hope this isn't too bad.

I was told that a vote was required for this sort of thing here.

Long story short, I picked this song because this server is pretty laid back compared to the others and I felt that a "sunset by the beach" sort of theme would fit this server perfectly.

Also, I REALLY hope that this worked right for this is my first pr.

### CODE REVIEW

**Comment by yellowface7** [comment]
If I did not do this right, here is a link to the video that is of my file I made.

https://www.youtube.com/watch?v=9VFenNH1Eoo

**Comment by alexkar598** [comment]
While it would be awesome for another game,I do not think it would fit in ss13 general theme.

**Review by alexkar598 — DISMISSED** [review]
Put the sound in sunset/sound/ambience/title.ogg and add the path to `strings/round_start_sounds.txt`
removing the other ones if you really want to.

**Comment by ghost** [comment]
You don't need to do that, you just need to put it in config/title_music/sounds or whatever

**Comment by yellowface7** [comment]
Later I will try to change the files. I just thought that it would fit this server. I had other candidates but they were too depressing.

**Comment by SmokingKilz** [comment]
Any song can fit SS13, we're going for a very specific theme with sunset and I think this would actually really fit that. Personally, I support this.

**Comment by yellowface7** [comment]
I will allocate time for development tomorrow around noon ish. I will try to replace the other song in that config directory. I already placed the song in the directory. This will be one of the three songs I have hand picked out and no, the other songs are not on my youtube channel. Also, what is with Travis? How do I make it happy? I find it funny how just simply replacing the old title works just as well as adding the name to the config list. The only weird thing is that during my tests, ONLY this song played. I need some help with fixing that, or does the config file fix that? My music overhaul is separate to this pr so keep an eye out on the pr for it. If there is anything else needed related to this pr, please let me know.

**Comment by alexkar598** [comment]
@yellowface7 ignore the errored Travis job,Travis is a mean boi

**Comment by yellowface7** [comment]
Okay. I added the file to the config folder. How do I update it on this pr?
